app/addons/documents/changes/components.react.jsx

```diff
@@ -351,6 +360,18 @@ define([
       };
     },
 
+    onClipboardClick: function (target) {
+      var msg = 'The document ID has been copied to your clipboard.';
```

Comment:
True, there's lot of instances where let will be better. Just wondering what you mean here though: wouldn't they be the same in this usage? I need msg to be defined in the function block to allow overriding. If I needed the var only in the `if` I could define it as a `let`, but I don't.